packages/sanes-chrome-extension/src/routes/signup/components/ShowPhraseForm.tsx

```diff
@@ -11,7 +11,6 @@ import { getGlobalPersona } from '../../../logic/persona';
 
 export const SECOND_STEP_SIGNUP_ROUTE = `${SIGNUP_ROUTE}2`;
 
-// Exported only for being used in test environments
```

Comment:
Do not remove this comment, right now is an escape solution for getting the mnemonic, but this will change soon. Info is going to be injected via the context API. For having a better visual alert, change it to:

// FIXME Refactor it once we inject data using Context API